setup.py

```diff
@@ -40,28 +42,27 @@
 # 'jmespath==0.9.3',
 # 'pyyaml==3.13'
 # 'knack>=0.3.1'
+# 'jsonschema~=3.0.2'
 
 # There is also a dependency for uamqp for amqp based commands
 # though that is installed out of band (managed by the extension)
 # for compatibility reasons.
 
-DEPENDENCIES = [
-    'paho-mqtt==1.3.1'
-]
+DEPENDENCIES = ["paho-mqtt==1.3.1", "jsonschema~=3.0.2"]
 
 
 setup(
-    name='azure-cli-iot-ext',
+    name="azure-cli-iot-ext",
     version=VERSION,
-    description='Provides the data plane command layer for Azure IoT Hub, IoT Edge and IoT Device Provisioning Service',
-    long_description='Authoritative IoT data plane extension for Azure CLI. '
-    'Focused on providing data plane commands for Azure IoT Hub, IoT Edge and IoT Device Provisioning Service',
-    license='MIT',
-    author='Microsoft',
-    author_email='iotupx@microsoft.com',  # +@digimaun
-    url='https://github.com/azure/azure-iot-cli-extension',
+    description="Provides the data plane command layer for Azure IoT Hub, IoT Edge and IoT Device Provisioning Service",
+    long_description="Authoritative IoT data plane extension for Azure CLI. "
+    "Focused on providing data plane commands for Azure IoT Hub, IoT Edge and IoT Device Provisioning Service",
+    license="MIT",
+    author="Microsoft",
+    author_email="iotupx@microsoft.com",  # +@digimaun
+    url="https://github.com/azure/azure-iot-cli-extension",
     classifiers=CLASSIFIERS,
-    packages=find_packages(),
-    package_data={'azext_iot': ['azext_metadata.json', 'digicert.pem']},
-    install_requires=DEPENDENCIES
+    packages=find_packages(exclude=["tests", "tests.*", "scripts"]),
+    package_data={"azext_iot": ["azext_metadata.json", "digicert.pem", "assets/edge-deploy-2.0.schema.json"]},
+    install_requires=DEPENDENCIES,
```

Comment:
another trailing comma, unless it is intentional

Reply:
Yea the trailing commas are coming from auto-formatting using Python Black...it's an amazing tool.